pkg/apis/configuration/validation/validation.go

```diff
@@ -96,7 +96,7 @@ func validateTime(time string, fieldPath *field.Path) field.ErrorList {
 }
 
 // http://nginx.org/en/docs/syntax.html
-const sizeFmt = `\d+[kKmM]?`
+const sizeFmt = `\d+[kKmMgG]?`
 const sizeErrMsg = "must consist of numeric characters followed by a valid size suffix. 'k|K|m|M"
```

Comment:
```suggestion
const sizeErrMsg = "must consist of numeric characters followed by a valid size suffix. 'k|K|m|M|g|G'"
```
Please update before merging